.changeset/twelve-actors-train.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/ui': major
```

Comment:
This should be a minor release, since it adds a new feature:
```suggestion
'@aws-amplify/ui': minor
```

Reply:
You're right